components/user/UserMilestone.js

```diff
@@ -23,8 +23,8 @@ export default function UserMilestone({ milestone, isGoal, manage }) {
       : "text-primary-medium dark:text-primary-low-medium";
 
     return (
-      <div className="flex space-x-3 grow">
-        {milestone.icon && <DisplayIcon className="h-8 w-8 rounded-full" />}
+      <div className="flex space-x-2 grow">
+        {milestone.icon && <DisplayIcon className="h-8 w-8 ml-11"/>}
```

Comment:
These changes are not part of this pull request plus break another page where it is used, please remove these changes so we can merge your pull request

```suggestion
      <div className="flex space-x-3 grow">
        {milestone.icon && <DisplayIcon className="h-8 w-8 rounded-full" />}
```

Reply:
okk. I'll discard changes suggested.👍